lib/omnibus/git_cache.rb

```diff
@@ -196,7 +196,11 @@ def remove_git_dirs
     # @return [Mixlib::Shellout] the underlying command object.
     #
     def git_cmd(command)
-      shellout!("git -c core.autocrlf=false --git-dir=#{cache_path} --work-tree=#{install_dir} #{command}")
+      shellout!("git -c core.autocrlf=false \
+                     -c core.ignorecase=false \
+                     --git-dir=#{cache_path} \
+                     --work-tree=#{install_dir} \
+                     #{command}".gsub(/\s+/, " "))
```

Comment:
Could this gsub mangle a user-provided install_dir that had more than 1 pace in its path? Is that a case we care about?